packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -70,6 +71,14 @@ async function run(options: {
   if (!fs.existsSync(srcDir)) {
     throw new Error(`--source path does not exist: ${srcDir}.`);
   }
+  const persistModule = options.persist && path.resolve(process.cwd(), options.persist);
+  if (persistModule && !fs.existsSync(persistModule)) {
+    throw new Error(`--persist path does not exist: ${persistModule}.`);
+  }
+  // Using eval here to prevent webpack from trying to rewrite the require and
+  // failing.
+  const persistQuery = persistModule &&
+    eval(`require(${JSON.stringify(persistModule)})`);
```

Comment:
In vanilla node, there is not (that's a Jest thing). I looked on the interwebs, but I couldn't find anything. Moving to webpack 2 would provide us with some more options, I think, but that's obviously a bigger change.